velox/exec/Task.cpp

```diff
@@ -1880,6 +1881,7 @@ ContinueFuture Task::terminate(TaskState terminalState) {
     numRunningDrivers_ = 0;
     for (auto& driver : drivers_) {
       if (driver) {
+        driversClosedByTask_.emplace_back(driver);
```

Comment:
Let's move inside the if
```
if (enterForTerminateLocked(driver->state()) ==
            StopReason::kTerminate)
```

Reply:
@xiaoxmeng
I don't think it is a good idea. There are some conditions that prevent drivers from being dded to offThreadDrivers.
I believe it is important to add such drivers too.

Reply:
ditto